### ISSUE TITLE
[Bug]: cargo-shuttle login panics

### ISSUE THREAD

**Issue by ad-ops** [shuttle-hq/shuttle#1300]
### What happened?

Running `cargo shuttle login --api-key=*******` or just `cargo shuttle login` and entering a valid key will panic the program.

Looks like the client has not been created correctly and it panics because of an `unwrap`. https://github.com/shuttle-hq/shuttle/blob/faaf9e88ee66475eacaaba033bcbcd5ea926f195/cargo-shuttle/src/lib.rs#L526C10-L526C10

Other commands without login seem to work such as `cargo shuttle project status --name ad-ops`.

### Version

v0.28.0

### Which operating system(s) are you seeing the problem on?

macOS

### Which CPU architectures are you seeing the problem on?

ARM64

### Relevant log output

```sh
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /Users/mork/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-shuttle-0.28.0/src/lib.rs:526:30
```

### Duplicate declaration

- [X] I have searched the issues and there are none like this.

**Comment by jonaro00** [comment]
Thanks for reporting! I'll fix this.

**Comment by orhun** [comment]
I also hit this: https://github.com/orhun/rustypaste/actions/runs/6410492324/job/17403900338

**Comment by jonaro00** [comment]
If you want to get logged in right now, you can do the `cargo shuttle init` command, which will also prompt you for logging in, but that won't panic.

**Comment by oddgrd** [comment]
@ad-ops thanks for the report! The fix has been released in cargo-shuttle v0.28.1

**Comment by ad-ops** [comment]
Thanks! Works now:-)
